src/sentry/static/sentry/app/views/organizationGroupDetails/utils.tsx

```diff
@@ -101,8 +115,8 @@ export function getSubscriptionReason(group, removeLinks = false) {
       );
     }
 
-    if (SUBSCRIPTION_REASONS.hasOwnProperty(reason)) {
-      return SUBSCRIPTION_REASONS[reason];
+    if (SUBSCRIPTION_REASONS.hasOwnProperty(reason as string)) {
+      return SUBSCRIPTION_REASONS[reason as string];
```

Comment:
```suggestion
    if (reason && SUBSCRIPTION_REASONS.hasOwnProperty(reason)) {
      return SUBSCRIPTION_REASONS[reason];
```

I think this will let you get rid of the cast.